ui/packages/consul-ui/app/styles/layout.scss

```diff
@@ -53,21 +53,30 @@ html[data-route$='edit'] .app-view > header + div > *:first-child {
 %app-view-content .container {
   margin-top: 1.25em;
 }
-.consul-upstream-instance-list,
-.consul-lock-session-list {
+
+%list-after-secondary-nav {
+  margin-top: 0 !important;
+}
+%list-after-secondary-nav ul {
+  border-top-width: 0 !important;
+}
+%list-after-filter-bar {
+  border-top-width: 0 !important;
+}
+
```

Comment:
Made it clearer what these styles are doing by naming them instead of just commenting. We can then use these as 'helper/utility' placeholders in individual components/routes until we can figure out a way not to need them at all.